tool/planet/main.go

```diff
@@ -217,8 +217,9 @@ func run() error {
 
 		cetcdInit = cetcd.Command("init", "Setup etcd to run the correct version").Hidden()
 
-		cetcdBackup     = cetcd.Command("backup", "Backup the etcd datastore to a file")
-		cetcdBackupFile = cetcdBackup.Arg("file", "The file to store the backup").Required().String()
+		cetcdBackup       = cetcd.Command("backup", "Backup the etcd datastore to a file")
+		cetcdBackupFile   = cetcdBackup.Arg("file", "The file to store the backup").String()
```

Comment:
Nit: Maybe mention that it's optional and will output to stdout if not specified.